src/data-browser/datasets/DocLink.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 
-function makeUrl({year, definition}) {
-  return `/documentation/${year}/data-browser-filters/#${definition}`
+function makeUrl({definition}) {
+  return `/documentation/tools/data-browser/data-browser-filters#${definition}`
```

Comment:
The direct links for filters no longer match now that we'd be using Docusaurus vs our own docs.
 
 Ex. From DB filtering (/data-browser/data/2021?category=states&items=AL&actions_taken=), clicking `Action Taken` to view documentation links to `/documentation/tools/data-browser/data-browser-filters#action_taken` but needs to be `/documentation/tools/data-browser/data-browser-filters#action-taken-action_taken`
 
Probably need to update this `buildWithId` function to include a new field for the documentation ID and use that value in this `makeUrl` function.

Reply:
@meissadia Great catch! I have implemented a solution. When going to data browser and clicking `Ethnicity` it would route you to `/documentation/tools/data-browser/data-browser-filters#ethnicity-derived_ethnicity`.

```JavaScript
import React from 'react'

function makeUrl(id, definition) {
  id = id.replace(' ', '-').toLowerCase()
  return `/documentation/tools/data-browser/data-browser-filters#${id}-${definition}`
}

const DocLink = props => {
  let id = props.children.props.children
  return (
    <a href={makeUrl(id, props.definition)} target="_blank" rel="noopener noreferrer">
      {props.children}
    </a>
  )
}

export default DocLink
```

This change has been deployed to the dev environment. Please view the change and let me know if this is a good working solution!

Reply:
This has been resolved.